app/src/main/java/com/mobile/tandil/javabaseproject/mvp/presenter/ParkingPresenter.java

```diff
@@ -6,10 +6,18 @@ public class ParkingPresenter implements ParkingContract.Presenter {
 
     private final ParkingContract.Model model;
     private final ParkingContract.View view;
+    private final String parkingAvailableString;
 
     public ParkingPresenter(ParkingContract.Model model, ParkingContract.View view) {
         this.model = model;
         this.view = view;
+        this.model.setParkingAvailable(30);
```

Comment:
this shouldn't be in the constructor. By the moment you can set it in the "onShowButtonPressed" method

Reply:
fixed